.github/CODEOWNERS

```
@@ -1,3 +1,4 @@
-experimenter/tests @jrbenny35
-experimenter/experimenter/ @jaredlockhart @yashikakhurana @eliserichards @brennie
 cirrus/ @jaredlockhart @yashikakhurana @jeddai
+experimenter/experimenter/ @jaredlockhart @yashikakhurana @eliserichards @brennie
+experimenter/schemas/ @jaredlockhart @mikewilli
```

I think this should just be `schemas/`?
```suggestion
schemas/ @jaredlockhart @mikewilli
```